scripts/preregcontent.csv

```diff
@@ -0,0 +1,73 @@
+LABEL,TEXT
```

Comment:
@leb2dg, it might be nice to create a subdirectory in scripts to contain all prereg related scripts/files.